examples/rainier/persist/rainier.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/persist as _;
+import ballerina/time;
+
+type Building record {|
+    readonly string buildingCode;
+    string city;
+    string state;
+    string country;
+    string postalCode;
+    string 'type;
+
+    Workspace[] workspaces;
+|};
+
+type Department record {|
+    readonly string deptNo;
+    string deptName;
+
+    Employee[] employees;
+|};
+
+type Employee record {|
+    readonly string empNo;
+    string firstName;
+    string lastName;
+    time:Date birthDate;
+    string gender;
+    time:Date hireDate;
+
+    Department department;
+    Workspace workspace;
+|};
+
+type OrderItem record {|
+    readonly string orderId;
+    readonly string itemId;
+    int quantity;
+    string notes;
+|};
+
+type Workspace record {|
+    readonly string workspaceId;
+    string workspaceType;
+
+    Building location;
+    Employee employee;
+|};
```

Comment:
```suggestion
|};

```

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)